legend-pure-runtime-java-engine-compiled/src/main/java/org/finos/legend/pure/runtime/java/compiled/generation/JavaSourceCodeGenerator.java

```diff
@@ -532,10 +460,10 @@ private String buildPureCompiledLambda(ProcessorContext processorContext)
                 "    public String __id(){return this.lambdaFunction == null ? \"Anonymous_Lambda\" : this.lambdaFunction.getName();}\n" +
                 "    public PureCompiledLambda copy()\n" +
                 "    {\n" +
-                "        PureCompiledLambda p =  new PureCompiledLambda();\n" +
-                "        p.lambdaFunction = (LambdaFunction)((AbstractCoreInstance)this.lambdaFunction).copy();\n" +
-                "        p.pureFunction = this.pureFunction;\n" +
-                "        return p;\n" +
+                "        return new PureCompiledLambda(" +
+                "                   (LambdaFunction)((AbstractCoreInstance)this.lambdaFunction).copy(),\n" +
+                "                   this.pureFunction" +
+                "                   );\n" +
```

Comment:
Adding newlines to go with the indentation.
```suggestion
                "        return new PureCompiledLambda(\n" +
                "                   (LambdaFunction)((AbstractCoreInstance)this.lambdaFunction).copy(),\n" +
                "                   this.pureFunction\n" +
                "                   );\n" +
```